autodispose/src/test/java/com/uber/autodispose/AutoDisposePluginsTest.java

```diff
@@ -17,44 +17,40 @@
 package com.uber.autodispose;
 
 import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
 import static com.google.common.truth.Truth.assertThat;
 
 public final class AutoDisposePluginsTest {
 
-  @After public void tearDown() {
+  @Before @After public void tearDown() {
```

Comment:
Same as other tearDown comment

Reply:
Part of 53772ce359b7db21c15342b6edc1ff3d809e35c3